openstudiocore/src/utilities/core/Path.cpp

```diff
@@ -129,9 +130,27 @@ path toPath(const char* s)
 path toPath(const std::string& s)
 {
   #ifdef Q_OS_WIN
-    std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>,wchar_t> converter;
-    std::wstring wstr = converter.from_bytes(s);
-    return path(wstr);
+    std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+	try {
+		std::wstring wstr = converter.from_bytes(s);
+		return path(wstr);
+	} catch (std::range_error& e) {
+		// Fallback to basically decoding latin-1
+		// Masking with 0xFF reduces any negative values into the range 0-255.
+		// TODO: another option is to write a codecvt_latin1 class
+		// cf: https://stackoverflow.com/questions/49669048/why-mask-a-char-with-0xff-when-converting-narrow-string-to-wide-string#49669048
```

Comment:
@macumber FYI

Reply:
@jmarrec what was happening? Was the previous code throwing an error?

Reply:
Yes, was throwing a `std::range_error` in `converter.from_bytes`